radixconfig.yaml

```diff
@@ -7,6 +7,9 @@ spec:
     - name: dev
       build:
         from: DEV
+    - name: dev-graph-poc
+      build:
+        from: 446-migrate-to-reactflow
```

Comment:
`from` is the tag that Radix will listen to pushes to and build the latest commit on that tag. This should probably be changed to DEV-GRAPH-POC. Should probably also add the following script to `package.json`:
- `release-dev-graph-poc`
- `use-dev-graph-poc`
- `run-dev-graph-poc`

Reply:
I'll add the scripts. Why would we change "from" to DEV-GRAPH-POC? From what I understand from reading [this](https://www.radix.equinor.com/references/reference-radix-config/#build-2), "from" is being used to deploy automatically from the specified branch and 446-migrate-to-reactflow is the branch where this is being developed.

Reply:
If you want every commit to be built automatically from that branch you can keep it like this, but once you delete the branch you will have to update the `radixconfig.yaml`. If you change it to build once the tag `DEV-GRAPH-POC` is updated you can use that environment when tagging from whatever branch you want.

Reply:
Alright. I'll keep it since I'm curious to try this method and we'll have to update the files when this is no longer a poc anyways.